float.css

```diff
@@ -1,6 +1,7 @@
 *{
```

Comment:
It is not good to use universal selector. Please remove this selector and have specific element or class you would like to apply to.